warehouse/templates/manage/release.html

```diff
@@ -117,7 +117,7 @@ <h3>Delete Release</h3>
 
   <div id="delete-release-modal" class="modal">
     <div class="modal__content" role="dialog">
-      <form method="POST" class="modal__form">
+      <form method="POST" class="modal__form" action="{{ request.route_path('manage.project.release', project_name=project.name, version=release.version) }}">
```

Comment:
Aha! This makes sense. We can use `action="{{ request.current_route_path() }}"` here instead.

Reply:
updated, thanks :)